src/test/java/seedu/address/logic/parser/AddTuitionTaskCommandParserTest.java

```diff
@@ -0,0 +1,92 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_DATE_TIME;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_DURATION;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_DATE_TIME;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_DURATION;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_EMPTY_TASK_DESC;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_NAME_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TASK_DESC;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TASK_WITHOUT_DESC;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_TASK_WITH_DESC;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import org.junit.Test;
+
+import seedu.address.logic.commands.AddTuitionTaskCommand;
+import seedu.address.model.person.TuitionTask;
+
+public class AddTuitionTaskCommandParserTest {
+    private AddTuitionTaskCommandParser parser = new AddTuitionTaskCommandParser();
+    private DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd/mm/uuuu HH:mm");
+
+    @Test
+    public void parse_validArgs_success() {
+        // With description
+        LocalDateTime taskDateTime = LocalDateTime.parse(VALID_DATE_TIME, formatter);
+        TuitionTask tuition = new TuitionTask("dummy", taskDateTime, VALID_DURATION, VALID_TASK_DESC);
+        assertParseSuccess(parser, "1 " + VALID_TASK_WITH_DESC,
+               new AddTuitionTaskCommand(tuition, INDEX_FIRST_PERSON));
+
+        // Without description
+        tuition = new TuitionTask(VALID_NAME_AMY, taskDateTime, VALID_DURATION, VALID_EMPTY_TASK_DESC);
```

Comment:
VALID_NAME_AMY cannot be used since the current implementation always uses "dummy"